packages/components/src/components/NumberShower.tsx

```diff
@@ -92,6 +94,7 @@ export const NumberShower: React.FC<NumberShowerProps> = ({
           </span>
         </span>
       ) : null}
+      {unitName && <span> {unitName + "s"}</span>}
```

Comment:
Going to be a problem with `foots` and other exceptions.

Reply:
(Ok, I see you don't use units yet for anything other than durations)

Reply:
Yea, good point. I'll add a comment here. I guess as I think about it though, "ms" is already kind of plural - will change.